packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -184,6 +184,15 @@ export class CdkErrorMapper {
       errorName: 'FilePermissionsError',
       classification: 'ERROR',
     },
+    {
+      errorRegex:
+        /EPERM: operation not permitted, rename (?<fileName>.*) → 'C:(.*)synth.lock(.*)/,
+      humanReadableErrorMessage: 'Not permitted to rename file: {fileName}',
+      resolutionMessage:
+        'Check that you have the right permissions to rename this file and try running the command again',
```

Comment:
These `EPERM` on Windows are likely related to concurrent access.
See https://www.google.com/search?q=eperm+windows+concurrent+access+to+file&rlz=1C5GCCM_en&oq=eperm+windows+concurrent+access+to+file&gs_lcrp=EgZjaHJvbWUyBggAEEUYOdIBCDY2NjRqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8 .

it could be permissions but it's more likely concurrent access and simple re-try may resolve it.

Could you please update the message to convey that possibility?